CMakeLists.txt

```diff
@@ -133,12 +133,14 @@ option(NGRAPH_CODE_COVERAGE_ENABLE "Enable code coverage data collection" FALSE)
 option(NGRAPH_LIB_VERSIONING_ENABLE "Enable shared library versioning" FALSE)
 option(NGRAPH_PYTHON_BUILD_ENABLE "Enable build nGraph python package wheel" FALSE)
 option(NGRAPH_PLAIDML_ENABLE "Enable the PlaidML backend" ${PLAIDML_FOUND})
+option(NGRAPH_INFERENCE_ENGINE_ENABLE "Enable the Inference Engine backend" FALSE)
 option(NGRAPH_FAST_MATH_ENABLE "Enable fast math" ON)
 option(NGRAPH_JSON_ENABLE "Enable JSON based serialization and tracing features" TRUE)
 option(NGRAPH_STATIC_LIB_ENABLE "Enable build nGraph as a static library" FALSE)
 option(NGRAPH_INTERPRETER_STATIC_LIB_ENABLE "Enable build INTERPRETER backend as a static library" FALSE)
 option(NGRAPH_CPU_STATIC_LIB_ENABLE "Enable build CPU backend as a static library" FALSE)
 option(NGRAPH_PLAIDML_STATIC_LIB_ENABLE "Enable build PlaidML backend as a static library" FALSE)
+option(NGRAPH_IE_STATIC_LIB_ENABLE "Enable build Inference Engine backend as a static library" FALSE)
```

Comment:
I know the name would be really long but it really should be NGRAPH_INFERENCE_ENGINE_STATIC_LIB_ENABLE, or we could rename the other variable as NGRAPH_IE_ENABLE, but either way they should match.

Reply:
I don't want to see `NGRAPH_CENTRAL_PROCESSING_UNIT_STATIC_LIB_ENABLE` so let's consistently go with `IE` instead of `INFERENCE_ENGINE`

Reply:
Good to hear that `IE` is enough :) I wasn't sure whether this would be sufficient and self-explanatory or whether introduce confusion what kind of backend it is.